<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QWidget" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auth</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="email_line_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_line_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="generate_token_button">
     <property name="text">
      <string>Register And Get Token</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="get_token">
     <property name="text">
      <string>Get Token</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Token:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="token_text_edit">
     <property name="placeholderText">
      <string>Already have token? Paste it here and sign in</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sign_in_button">
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
